MIGRATION.md

```diff
@@ -70,37 +75,93 @@
 
 ## How to migrate your packages to v4
 
-Install a package that contains all of the components from Forma 36. Tree-shaking will take care of your build, so it will include only components that you use.
-We separate only icons from the main package. If you need to use our icons, install them separately, by running the following commands:
-:
+### Step 1 - install new packages
```

Comment:
Potentially we will have just one codemod command to do everything. We still would need to have this kinda step by step guide anyway.